examples/genomicsdb_cache

```diff
@@ -0,0 +1,119 @@
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+import argparse
+import os
+
+import genomicsdb
+
+
+def normalize_path(path):
+    if "://" in path:
+        return path
+    else:
+        return os.path.abspath(path)
+
+
+def is_cloud_path(path):
+    if "://" in path:
+        return True
+    else:
+        return False
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        prog="cache",
+        description="Cache GenomicsDB generated json artifacts for workspace cloud URLs",
+        formatter_class=argparse.RawTextHelpFormatter,
+        usage="%(prog)s [options]",
+    )
+    parser.add_argument(
+        "--version",
+        action="version",
+        version=genomicsdb.version(),
+        help="print GenomicsDB native library version and exit",
+    )
+    parser.add_argument(
+        "-w",
+        "--workspace",
+        required=True,
+        help="URL to GenomicsDB workspace \ne.g. -w my_workspace or -w az://my_container/my_workspace"
+        + " or -w s3://my_bucket/my_workspace or -w gs://my_bucket/my_workspace",
+    )
+    parser.add_argument(
+        "-v",
+        "--vidmap",
+        required=False,
+        help="Optional - URL to vid mapping file. Defaults to vidmap.json in workspace",
+    )
+    parser.add_argument(
+        "-c",
+        "--callset",
+        required=False,
+        help="Optional - URL to callset mapping file. Defaults to callset.json in workspace",
+    )
+    parser.add_argument(
+        "-l", "--loader", required=False, help="Optional - URL to loader file. Defaults to loader.json in workspace"
+    )
+
+    args = parser.parse_args()
+
+    workspace = normalize_path(args.workspace)
+    if not genomicsdb.is_file(workspace + "/__tiledb_workspace.tdb"):
+        raise RuntimeError(f"workspace({workspace}) not found")
+
+    callset_file = args.callset
+    if not callset_file:
+        callset_file = workspace + "/callset.json"
+    vidmap_file = args.vidmap
+    if not vidmap_file:
+        vidmap_file = workspace + "/vidmap.json"
+    loader_file = args.loader
+    if not loader_file:
+        loader_file = workspace + "/loader.json"
+    if (
+        not genomicsdb.is_file(callset_file)
+        or not genomicsdb.is_file(vidmap_file)
+        or not genomicsdb.is_file(loader_file)
+    ):
+        raise RuntimeError(f"callset({callset_file}) vidmap({vidmap_file}) or loader({loader_file}) not found")
+
+    if is_cloud_path(callset_file):
+        with open("callset.json", "wb") as f:
+            f.write(genomicsdb.read_entire_file(callset_file).encode())
+    if is_cloud_path(vidmap_file):
+        with open("vidmap.json", "wb") as f:
+            f.write(genomicsdb.read_entire_file(vidmap_file).encode())
+    if is_cloud_path(loader_file):
+        with open("loader.json", "wb") as f:
+            f.write(genomicsdb.read_entire_file(loader_file).encode())
```

Comment:
Is this sufficient? The loader can have pointers to the callset and vid file right? Wouldn't we need to update those for this to work?

Reply:
So far, I have used the loader file only to figure out partitions and is not passed down to GenomicsDB. The callset and vidmap files should be in the workspace if we used `vcf2genomicsdb_init` and that is what is being used as a default. I could use the values from the loader file as the default, would that be preferable for consistency?

Reply:
I see -- okay, I think its fine to keep it as is.